luajava/src/main/java/party/iroiro/luajava/Lua.java

```diff
@@ -759,6 +759,19 @@ public interface Lua extends AutoCloseable {
      */
     void createTable(int nArr, int nRec);
 
+    /**
+     * Creates a new empty table and pushes it onto the stack
+     * 
```

Comment:
Missing a `<p>` tag.

Reply:
> Missing a `<p>` tag.

As far as i see at other docs string, leading one line are not inside <p> tag and following lines are inside <p> tag. 

Maybe I just confused the rule but I cannot found where are the specs located. 

Can you mention specific example or the document of docs/code formatting please?

Reply:
Well, the comment was intended for these lines:
https://github.com/gudzpoz/luajava/blob/4dfecbbda5dbc53e7fb307b9d9a846676410b37a/luajava/src/main/java/party/iroiro/luajava/Lua.java#L764-L766

The current javadoc looks good to me.